CHANGELOG.md

```diff
@@ -0,0 +1,290 @@
+[Fixes display bug where drop down user menu covered user's avatar](https://github.com/UW-Madison-DoIT/uw-frame/pull/422)
+[Fixes display bug in user drop down menu](https://github.com/UW-Madison-DoIT/uw-frame/pull/425)
+
+## [4.0.3][] - 2017-04-13
+
+[Fixes a bug where rss widgets would always show an error](https://github.com/UW-Madison-DoIT/uw-frame/pull/397)
+
+## [4.0.2][] - 2017-04-13
+
+Bug fixes for a specific widget (leave balances) #395 #394
+
+## [4.0.1][] - 2017-04-13
+
+This release has some code cleanup and some bug fixes.  Upgrade from [v4.0.0](https://github.com/UW-Madison-DoIT/uw-frame/releases/tag/uw-frame-maven-4.0.0) by only updating the dependency version.
+
+[Cleans up the old beta header announcement](https://github.com/UW-Madison-DoIT/uw-frame/pull/388/files).  The hard coded announcement was before the feature of priority notifications.
+[Ensures consistent behavior on hover](https://github.com/UW-Madison-DoIT/uw-frame/pull/390)
+[Ensures consistent link launch behavior in widgets](https://github.com/UW-Madison-DoIT/uw-frame/pull/391)
+[Fixes bug where icons weren't showing when using the widget directive](https://github.com/UW-Madison-DoIT/uw-frame/pull/393)
+
+## [4.0.0][] - 2017-04-05
+
+This release adds a widget directive to the app-framework, adds personalized notifications, progresses [Apereo](https://www.apereo.org/) [Incubation](https://www.apereo.org/incubation), and fixes a help url bug.
+
+[Guides Contributors to Apereo CLA compliance](https://github.com/UW-Madison-DoIT/uw-frame/pull/381)
+[Changes help url for uw-system schools](https://github.com/UW-Madison-DoIT/uw-frame/pull/382)
+[Allows widgets to be embedded in frame apps](https://github.com/UW-Madison-DoIT/uw-frame/pull/385)
+[Adds widget directive documentation](https://github.com/UW-Madison-DoIT/uw-frame/pull/384)
+[Adds personalized notifications](https://github.com/UW-Madison-DoIT/uw-frame/pull/386)
+[Fixes bug in personalized notifications](https://github.com/UW-Madison-DoIT/uw-frame/pull/387)
+
+### How to Upgrade from version 3.X
+* Change your dependency to version 4.0.0
+* If you were consuming notifications within the app, the [notification service api changed from `getNotificationsByGroups` to `getFilteredNotifications`](https://github.com/UW-Madison-DoIT/uw-frame/pull/386/files#diff-9a244329b1c0f99008f0fb506d3b9c64L159).  This will not be a concern for most (if not all) applications.
+* If you had previously named services and directives in your MyUW app with the word widget in the name, you may run into some naming conflicts due to the new Widget directives and services being added.  [See the code for more details](https://github.com/UW-Madison-DoIT/uw-frame/pull/385/files).  This will not be a concern for most applications.
+
+## [3.1.4][] - 2017-03-20
+
+Fixes a bug where text in a static portlet was hard to read (#380)
+
+## [3.1.3][] - 2017-03-09
+
+Adds back in UW-Lacrosse theming
+
+## 3.1.2 - \[Yanked]
```

Comment:
:question: any details on why this release was skipped or yanked that could be added?

Reply:
My guess is that something went wrong with the release to Maven central, so the git tag was yanked, and the release process tried again.

I'm not 100% sure, but I'm 95% confident that's what happened.